src/regions/ca/info/ca-nl/clinics.fr.md

```diff
@@ -6,7 +6,7 @@ Vous pouvez effectuer cette évaluation pour vous-même ou pour le compte d'une
 
 Si vous avez des questions sur COVID-19, veuillez consulter les ressources suivantes :
 
-- gov.nl.ca/covid-19 - Renseignez-vous sur COVID-19 et sur les mesures prises pour protéger la santé des Terre-Neuviens et des Labradoriens.
+- [gov.nl.ca/covid-19](https://www.gov.nl.ca/covid-19/fr/ressources/) - Renseignez-vous sur COVID-19 et sur les mesures prises pour protéger la santé des Terre-Neuviens et des Labradoriens.
```

Comment:
This could be formated nicer:

```suggestion
- [Renseignez-vous sur COVID-19 et sur les mesures prises pour protéger la santé des Terre-Neuviens et des Labradoriens](https://www.gov.nl.ca/covid-19/fr/ressources/)
```

Reply:
Hum vraiment bonne question. Je l'aurais aussi mis comme toi idéalement mais il était comme ça pour le gouvernement. Dans le bot, il prenait la phrase directement fait que le lien n'était pas cliquebale. C'est ce que le Dr Lalla a pointé. Je me suis dit qu'en mettant l'adresse cliquebale, ça fixerait le problème. Je ne sais pas si ça pose problème si on enlève l'adresse dans le texte, est-ce que le bot va être capable de savoir que dernière ce lien ce sont des renseignements du covid. Je vais me mettre une note pour en jaser demain aussi. pourrait-on le mettre comme j'ai proposé pour qu'on voit si ça fix le problème ensuite dans le bot et ensuite on pourrait essayer de le rendre cute ?

Reply:
Yes, they bot will still see that it is a link :) It just makes the formating much nicer imo

Reply:
Nice, on le rend cute alors ! Pis je ferais le test pour voir si ça fix en même temps le problème de Dr Lalla ;)
merciii